src/main/java/com/redhat/devtools/intellij/quarkus/lsp4ij/LanguageServerWrapper.java

```diff
@@ -724,6 +724,10 @@ public String getLanguageId(Language language) {
     }
 
     void registerCapability(RegistrationParams params) {
+        // no-op if initializeFuture is null
+        if (initializeFuture == null) {
```

Comment:
@jeffmaury: Wondering if this fix is good enough: wouldn't the plugin end up without functionality if registration isn't retriggered?